src/components/side_panel/chart/building_blocks/series_design/series_design_editor.ts

```diff
@@ -33,20 +36,27 @@ export class SeriesDesignEditor extends Component<Props, SpreadsheetChildEnv> {
     updateChart: Function,
     canUpdateChart: Function,
     slots: { type: Object, optional: true },
+    canChooseAxis: { type: Boolean, optional: true },
   };
 
-  protected state = useState({ index: 0 });
+  get seriesOffset() {
+    return this.props.canChooseAxis ? 1 : 0;
+  }
+
+  protected state = useState({ index: -this.seriesOffset });
```

Comment:
How about getting rid of `seriesOffset` for some easy to understand boolean `cadEditAllSeries` and change `state.index: number` to `state.selectedSeries: "all" | number` ? 

I feel like index kinda sucks as a variable name (even more so if it is referenced outside the component) and having index === -1 being "all series selected" isn't helping the ease of understanding of this code IMO

Reply:
I agree. index -1 doesn't mean anything when reading the code. Let's make it more explicit